src/tests/topp/ProteinQuantifier_9_output_mztab.mzTab

```diff
@@ -50,14 +50,14 @@ PRH	accession	description	taxid	species	database	database_version	search_engine
 PRT	Protein	null	null	null	null	null	null	0.9097144245	null	null	null	null	null	null	null	null	null	null	null	null	null	null	null	protein_details
 PRT	Protein	null	null	null	null	null	null	0.9097144245	Protein	null	null	1230.0	230.0	1030.0	1230.0	null	null	230.0	null	null	1030.0	null	null	single_protein
 
-PEH	sequence	accession	unique	database	database_version	search_engine	best_search_engine_score[1]	search_engine_score[1]_ms_run[1]	search_engine_score[1]_ms_run[2]	search_engine_score[1]_ms_run[3]	modifications	retention_time	retention_time_window	charge	mass_to_charge	spectra_ref	peptide_abundance_study_variable[1]	peptide_abundance_stdev_study_variable[1]	peptide_abundance_std_error_study_variable[1]	peptide_abundance_study_variable[2]	peptide_abundance_stdev_study_variable[2]	peptide_abundance_std_error_study_variable[2]	peptide_abundance_study_variable[3]	peptide_abundance_stdev_study_variable[3]	peptide_abundance_std_error_study_variable[3]	opt_global_cv_MS:1000889_peptidoform_sequence
-PEP	AAAK	Protein	1	null	null	null	1.0	1.0	null	null	null	100.0	null	2	100.0	null	1000.0	null	null	null	null	null	1000.0	null	null	AAAK
-PEP	CCCK	Protein	1	null	null	null	1.0	1.0	null	null	null	200.0	null	2	200.0	null	200.0	null	null	200.0	null	null	null	null	null	CCCK
-PEP	EEEK	Protein	1	null	null	null	1.0	null	1.0	null	null	300.0	null	2	300.0	null	30.0	null	null	30.0	null	null	30.0	null	null	EEEK
-PEP	GGG	Protein	1	null	null	null	1.0	null	null	1.0	null	400.0	null	2	400.0	null	4.0	null	null	4.0	null	null	null	null	null	GGG
+PEH	sequence	accession	unique	database	database_version	search_engine	best_search_engine_score[1]	search_engine_score[1]_ms_run[1]	search_engine_score[1]_ms_run[2]	search_engine_score[1]_ms_run[3]	modifications	retention_time	retention_time_window	charge	mass_to_charge	spectra_ref	peptide_abundance_study_variable[1]	peptide_abundance_stdev_study_variable[1]	peptide_abundance_std_error_study_variable[1]	peptide_abundance_study_variable[2]	peptide_abundance_stdev_study_variable[2]	peptide_abundance_std_error_study_variable[2]	peptide_abundance_study_variable[3]	peptide_abundance_stdev_study_variable[3]	peptide_abundance_std_error_study_variable[3]	opt_global_id_merge_index	opt_global_cv_MS:1000889_peptidoform_sequence
+PEP	AAAK	Protein	1	null	null	null	1.0	1.0	null	null	null	100.0	null	2	100.0	null	1000.0	null	null	null	null	null	1000.0	null	null	null	AAAK
```

Comment:
Do you know why this is added, although all the values are null?

Reply:
I will check but I think this is a meta value added to the PeptideHit or Id during merging

Reply:
yes, but if all are null, it should not appear as a column, right? Either all IDs have that MV or none.

Reply:
could be a bug. I think it is a meta value at the PeptideIdentification level
https://github.com/OpenMS/OpenMS/blob/develop/src/tests/class_tests/openms/data/ProteinQuantifier_input.consensusXML#L32
and then read from the peptide hits. maybe we don't expose the peptideidentification meta values at all?

Reply:
Yes I remember that or a similar limitation